applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="y_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing y-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="z_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing z-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+            </group>
+        </group>
+        <group name="measurement" type="NXobject" optional="true">
+            <!--the choice if a concept filling content in NXfabrication is recommended or optional
+was made such that all for all those components which are typically add-ons in a
+microscope it is more likely that individuals will have bought different third-party tools
+and therefore, for these typically more optional add-ons NXfabrication is recommended
+for others commercial microscopes often come with model-specific predefined parts
+hence using optional is sufficient-->
+            <group name="instrument" type="NXinstrument_em">
+                <field name="name" type="NX_CHAR" recommended="true"/>
+                <field name="location" type="NX_CHAR" recommended="true"/>
+                <field name="type" type="NX_CHAR" recommended="true"/>
+                <group name="fabrication" type="NXfabrication">
+                    <field name="vendor" type="NX_CHAR"/>
+                    <field name="model" type="NX_CHAR"/>
+                    <field name="serial_number" type="NX_CHAR" recommended="true"/>
+                </group>
+                <group type="NXprogram" recommended="true">
+                    <doc>
+                        Details about the control program used for operating the microscope.
+                        
+                        Instances should use control_software as a name prefix.
```

Comment:
So should this field be named control_software_PROGRAM with nameType="partial"?  This comment applies to other parts of the definition with the pattern "Instances should use XXX as a name prefix."

Reply:
Yes, this suggestion has been accepted, in the new draft all instances of design pattern "nameType: any" with docstring "Instances should" have been replaced according to the above-mentioned suggestion as it was also discussed at the last telco